src/EventsPage/css/add-events.css

```diff
@@ -1,3 +1,14 @@
+ .add-event{
```

Comment:
I'd put a space between the selector and the brace, which seems to be our style so far.

Reply:
Here and elsewhere